qiskit_nature/problems/second_quantization/lattice/models/fermi_hubbard_model.py

```diff
@@ -0,0 +1,148 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The Fermi-Hubbard model"""
+from typing import Optional
+import numpy as np
+from qiskit_nature.problems.second_quantization.lattice.lattice import Lattice
+from qiskit_nature.operators.second_quantization import FermionicOp
+
+
+class FermiHubbardModel:
+    """The Fermi-Hubbard model."""
+
+    def __init__(self, lattice: Lattice, onsite_interaction: complex) -> None:
+        """
+        Args:
+            lattice: Lattice on which the model is defined.
+            onsite_interaction: The strength of the on-site interaction.
+        """
+        self._lattice = lattice
+        self.onsite_interaction = onsite_interaction
+
+    @property
+    def lattice(self) -> Lattice:
+        """Return a copy of the input lattice."""
+        return (self._lattice).copy()
+
+    def hopping_matrix(self) -> np.ndarray:
+        """Return the hopping matrix
+
+        Returns:
+            The hopping matrix.
+        """
+        return self.lattice.to_adjacency_matrix()
```

Comment:
```suggestion
        return self._lattice.to_adjacency_matrix()
```
Or does this require a copy first?